### PR TITLE
add email option when starting an analysis

### PR DESCRIPTION
**Pull request by robinandeer**
default to email of logged in user from web UI

Please review @ingkebil

### CODE REVIEW

File: moneypenny/store/models/user.py [hunk 1]
```diff
@@ -15,3 +15,7 @@ class User(Model, UserMixin):
     @property
     def first_name(self):
         return self.name.split(' ')[0]
+
+    @property
+    def email(self):
+        return self.id
```
Comment (ingkebil) [inline moneypenny/store/models/user.py]: The user id is their email address?

**Comment by ingkebil** [comment]
Looks good! :+1:

File: moneypenny/flow/actions/analyze.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ class GeneListConflictError(Exception):
 }
 
 
-def dispatch(db, case_id, pedigree=True):
+def dispatch(db, case_id, pedigree=True, email=None):
     """Dispatch a new analysis for a case."""
```
Comment (ingkebil) [inline moneypenny/flow/actions/analyze.py]: No doc strings for the arguments?

File: moneypenny/store/models/user.py (hunk 1 above)
Comment (robinandeer) [inline moneypenny/store/models/user.py]: Well I thought; why not? It's guaranteed to be unique and then I don't have to keep track of multiple user ids

Is it bad practice?

Comment (ingkebil) [inline moneypenny/store/models/user.py]: Oh no, I love it!

File: moneypenny/flow/actions/analyze.py (hunk 2 above)
Comment (robinandeer) [inline moneypenny/flow/actions/analyze.py]: Good call! Now that it's a limited PR I can add this 👍

**Pull request merged**